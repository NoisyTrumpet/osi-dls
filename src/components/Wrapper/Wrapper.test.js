import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Wrapper } from './';

describe('<Wrapper />', () => {
  test('Should render', () => {
    const wrapper = shallow(<Wrapper />);
    expect(wrapper.length).toBe(1);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
