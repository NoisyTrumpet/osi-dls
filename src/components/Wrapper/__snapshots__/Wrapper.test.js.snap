// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> Should render 1`] = `
<div
  className="wrapper"
/>
`;
